lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -36,81 +37,83 @@ if [ ! -f "$ROOTCA" ]; then
+  echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 
-cat > /etc/apt/apt.conf.d/20auto-upgrades << EOF
-APT::Periodic::Update-Package-Lists "0";
-APT::Periodic::Download-Upgradeable-Packages "0";
-APT::Periodic::AutocleanInterval "0";
-APT::Periodic::Unattended-Upgrade "0";
+  cat > /etc/apt/apt.conf.d/20auto-upgrades << EOF
+  APT::Periodic::Update-Package-Lists "0";
+  APT::Periodic::Download-Upgradeable-Packages "0";
+  APT::Periodic::AutocleanInterval "0";
+  APT::Periodic::Unattended-Upgrade "0";
 EOF
 
-apt purge --auto-remove unattended-upgrades -y
-apt-mark hold "$(uname -r)" linux-aws linux-headers-aws linux-image-aws
-
-
-# interface config
-INTERFACE_DIR="/etc/network/interfaces.d"
-mkdir -p "$INTERFACE_DIR"
-echo "source-directory $INTERFACE_DIR" > /etc/network/interfaces
-
-# get rid of netplan
-systemctl unmask networking
-systemctl enable networking
-
-echo "Install Magma"
-apt-get update -y
-apt-get upgrade -y
-apt-get install curl zip python3-pip docker.io net-tools sudo -y
-
-echo "Making sure $MAGMA_USER user is sudoers"
-if ! grep -q "$MAGMA_USER ALL=(ALL) NOPASSWD:ALL" /etc/sudoers; then
-  adduser --disabled-password --gecos "" $MAGMA_USER
-  adduser $MAGMA_USER sudo
-  echo "$MAGMA_USER ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
-fi
-
-adduser $MAGMA_USER docker
-
-alias python=python3
-pip3 install ansible
-
-rm -rf /opt/magma/
-git clone "${GIT_URL}" /opt/magma
-cd /opt/magma || exit
-git checkout "$MAGMA_VERSION"
-
-# changing intefaces name
-sed -i 's/GRUB_CMDLINE_LINUX=""/GRUB_CMDLINE_LINUX="net.ifnames=0 biosdevname=0"/g' /etc/default/grub
-sed -i 's/ens5/eth0/g' /etc/netplan/50-cloud-init.yaml
-# changing interface name
-update-grub2
-
-cat > /etc/netplan/70-secondary-itf.yaml << EOF
-network:
-    ethernets:
-        eth1:
-            dhcp4: true
-            dhcp6: false
-    version: 2
+  apt purge --auto-remove unattended-upgrades -y
+  apt-mark hold "$(uname -r)" linux-aws linux-headers-aws linux-image-aws
+
+  # interface config
+  INTERFACE_DIR="/etc/network/interfaces.d"
+  mkdir -p "$INTERFACE_DIR"
+  echo "source-directory $INTERFACE_DIR" > /etc/network/interfaces
+
+  # get rid of netplan
+  systemctl unmask networking
+  systemctl enable networking
+
+  echo "Install Magma"
+  apt-get update -y
+  apt-get upgrade -y
+  apt-get install curl zip python3-pip docker.io net-tools sudo docker-compose -y
+
+  echo "Making sure $MAGMA_USER user is sudoers"
+  if ! grep -q "$MAGMA_USER ALL=(ALL) NOPASSWD:ALL" /etc/sudoers; then
+    adduser --disabled-password --gecos "" $MAGMA_USER
+    adduser $MAGMA_USER sudo
+    echo "$MAGMA_USER ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+
+    adduser $MAGMA_USER docker
+  fi
+
+  alias python=python3
+  pip3 install ansible
+
+  rm -rf /opt/magma/
+  git clone "${GIT_URL}" /opt/magma
+  cd /opt/magma || exit
+  git checkout "$MAGMA_VERSION"
+
+  # changing intefaces name
```

Comment:
I agree that changing interface names helps in reducing other custom configurations, but it would be ideal to get away from fixed interface names and move to predictable interface names in the future since the name of NIC contains relevant information about the device and the network in general.